### PR TITLE
Issue 2

### PR DESCRIPTION
**Pull request by jtryan**
Marvin,
This should get you close to what you want. Some CSS work will change the display.  The only bug I found that I leave to you to address is that LocalStorage is being used to let the timer stay consistent as the student moves to the next question.  And it is not always cleared when the student ends the exam or changes pages.

Addresses #2

### CODE REVIEW

File: static/scripts/main.js [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+
+let h = "";
+let m = "";
+let s = "";
+// let newDateObj = new Date();
+// let oldDateObj = new Date();
+
+
+if(localStorage.getItem("secondsLeft") == null) {
+    let timer = document.getElementById("time").value;
```
Comment (MarvinKweyu) [inline static/scripts/main.js]: Hey @jtryan , where exactly did you set the `id='time'` in the `html` file again? I've tried setting a dummy `<h2>` just above the `timer` id but I still cannot get it to display.

**Comment by devops-john** [comment]
@MarvinKweyu 

take_quiz_form.html
line 6 is a hidden field to allow that information to be pulled into the js
'''html
<input type="hidden" id="time" name="timer" value="{{ quiz.duration }}">
'''

Then line 16 displays it.
'''html
<h2 id="timer"></h2>
'''

**Comment by MarvinKweyu** [comment]
Ah.. I see. HAd totally not seen that. Did you get it to show though? Perhaps I should re-run it on my end.

**Comment by devops-john** [comment]
Oh yes. If you run the branch it will display on the student test.

John Ryan
Cloud Engineer
FDB (First Databank, Inc.)
Direct: 317.571.7349
john.ryan@fdhbhealth.com<mailto:john.ryan@fdhbhealth.com>

Corporate website<https://www.fdbhealth.com/> | Follow us on Twitter<https://twitter.com/FDB_US> and LinkedIn<https://www.linkedin.com/company/firstdatabank> | FDB Insights<https://www.fdbhealth.com/insights>

PART OF THE HEARST HEALTH NETWORK

**Pull request merged**